ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -421,10 +449,18 @@ const OmimSegments = styled(Segment.Group).attrs({ size: 'tiny', horizontal: tru
   }
 `
 
-const getDetailSections = (configs, gene, compact, labelProps, rnaSeqData) => configs.map(
+const getDetailSections = (configs, gene, compact, labelProps, individualGeneData) => configs.map(
   ({ showDetails, detailsDisplay, ...sectionConfig }) => (
-    { ...sectionConfig, detail: showDetails(gene, rnaSeqData) && detailsDisplay(gene, rnaSeqData) }),
-).filter(({ detail }) => detail).map(({ detail, expandedDisplay, ...sectionConfig }) => (
+    {
+      ...sectionConfig,
+      detail: showDetails(gene, individualGeneData) && detailsDisplay(gene, individualGeneData),
+    }),
+).reduce((acc, config) => (Array.isArray(config.detail) ?
+  [
+    ...acc,
+    ...config.detail.map(([tool, detail]) => ({ ...config, label: tool.toUpperCase(), detail })),
+  ] : [...acc, config]),
+[]).filter(({ detail }) => detail).map(({ detail, expandedDisplay, ...sectionConfig }) => (
```

Comment:
you should do the filter before the reduce, or do the filtering as part of the reduce